packages/stockflux-components/src/components/preview-window/PreviewWindow.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import { StockFluxHooks } from 'stockflux-core';
+
+export default ({
+  htmlfile = 'transparent-window.html',
+  children,
+  position = { left: 100, top: 100 },
+  size = { width: 300, height: 300 },
+  display
+}) => {
+  const TRANSPARENT_WINDOW_NAME = 'transparent_window';
+  const TRANSPARENT_CSS_PATCH = 'transparentWindow.css';
+
+  const windowStatus = {
+    showing: 'SHOWING',
+    hiding: 'HIDING',
+    hide: 'HIDE',
+    hidden: 'HIDDEN'
+  };
+
+  const childWindow = StockFluxHooks.useChildWindow(
+    TRANSPARENT_WINDOW_NAME,
+    document,
+    TRANSPARENT_CSS_PATCH
+  );
+
+  const { window, launch } = childWindow;
+
+  const launchChildWindow = useCallback(
+    () =>
+      launch({
+        name: TRANSPARENT_WINDOW_NAME,
+        waitForPageLoad: true,
```

Comment:
The first time you drag the chart outside of the watchlist, there's a delay before it loads. The rest of the time it loads quickly, just wondering if this was the reason why and if it can load any quicker on the first load?

Reply:
This is because the first time the window has to appear it has to wait for the childwindow to be created and populated, which has a slight delay which I have tried to keep as small as possible by not having the popup have any dynamic or react components.
Every subsequent time it is just show/hiding the window so it loads faster.